### DISCUSSION TITLE
Help! Is the Next.js even compatible with the Docker Compose?

### DISCUSSION THREAD

**Discussion by markovmx** [vercel/next.js#63766] [Help]
### Summary

I’ve tried all the official and unofficial examples. It turns out either the site can not display pictures from api, gets error 500, or nothing works.

**Comment by Willem-Jaap** [comment]
Have you checked out the Next.js Docker compose example: https://github.com/vercel/next.js/tree/canary/examples/with-docker-compose

**Reply by markovmx** [reply]
Thanks for answer. But I wrote earlier official examples do not work in my case.

**Reply by Willem-Jaap** [reply]
Can you supply more information or a minimal reproduction?

**Reply by markovmx** [reply]
<img width="1728" alt="Screenshot 2024-03-27 at 16 28 08" src="https://github.com/vercel/next.js/assets/121501065/5ecbabf0-df99-49b9-8baa-fc784810aba2">

I have fullstack app (nextjs and golang) and launch it with docker-compose. As you can see on screenshot: i receive from my api string with version of database and display it. But when receive image from my api i got error 500. In both cases i use url: http://localhost:8080.

next.config.js

/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: false,
  images: { remotePatterns: [{ hostname: "localhost" }] },
};

export default nextConfig;

**Reply by Willem-Jaap** [reply]
I have no experience with go but I think the issue is not related to Next.js as the go service does not return a valid response.
Can you confirm you can access the image on http://localhost:8080/assets/images/banner/jpg ?
